velox/dwio/parquet/reader/duckdb/InputStreamFileHandle.h

```diff
@@ -22,33 +22,16 @@ namespace facebook::velox::duckdb {
 
 class InputStreamFileSystem;
 
+// Implements the DuckDB FileHandle API
 class InputStreamFileHandle : public ::duckdb::FileHandle {
```

Comment:
Isn't this essentially `using InputStreamFileHandle = ::duckdb::Filehandle` now ? Or is a separate inherited class somehow mandatory to use that class ?

Reply:
This and `duckdb::InputStreamFileSystem` both are needed to satisfy the DuckDB Parquet Reader API requirements.